sympy/functions/special/tests/test_error_functions.py

```diff
@@ -201,7 +201,7 @@ def test_erfi():
     assert erfi(x).as_leading_term(x) == 2*x/sqrt(pi)
     assert erfi(x*y).as_leading_term(y) == 2*x*y/sqrt(pi)
     assert (erfi(x*y)/erfi(y)).as_leading_term(y) == x
-    assert erfi(1/x).as_leading_term(x) == erfi(1/x)
+    assert erfi(1/x).as_leading_term(x) == 1/(x*sqrt(-1/x**2))
```

Comment:
Could this be expressed in terms of `sign(x)`?

Reply:
That would need the assumption that x is real. Should that be made?

Reply:
Actually, it seems that `erfi(x)` will be of the order of `exp(x**2)` as `x` tends to infinity so there will be no leading term for real `x`. For pure imaginary `x`, it is bounded (`-I*erf(I*x)`), but that should not considered here. So it looks like the call should raise.